File: tests/human/x/dart/match_expr.dart
void main() {
  var x = 2;
  var label;
  switch (x) {
    case 1:
      label = 'one';
      break;
    case 2:
      label = 'two';
      break;
    case 3:
      label = 'three';
      break;
    default:
      label = 'unknown';
  }
  print(label);
}

File: archived/tests/transpiler/x/java/match_expr.java
public class Main {
    static int x = 2;
    static String label = x == 1 ? "one" : x == 2 ? "two" : x == 3 ? "three" : "unknown";

    public static void main(String[] args) {
        System.out.println(label);
    }
}
